<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Geeta" failures="1" tests="2" name="Test" time="11.743" errors="0" timestamp="2025-06-19T12:22:09 IST">
  <testcase classname="com.testcases.TC_LoginPage" name="verifyLoginFailed" time="5.732">
    <failure type="java.lang.AssertionError" message="expected [Logged In Successfully1] but found [Logged In Successfully]">
      <![CDATA[java.lang.AssertionError: expected [Logged In Successfully1] but found [Logged In Successfully]
at com.testcases.TC_LoginPage.verifyLoginFailed(TC_LoginPage.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginFailed -->
  <testcase classname="com.testcases.TC_LoginPage" name="verifyLoginSucc" time="3.682"/>
  <testcase classname="com.testcases.TC_LoginPage" name="skippedTest" time="0.0">
    <ignored/>
  </testcase> <!-- skippedTest -->
</testsuite> <!-- Test -->
